src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,13 +45,23 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        Set<String> upperCaseKeywords = changeStringsToUpperCase(keywords);
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> wordsInName = changeStringsToUpperCase(new HashSet<>(person.getName().getWordsInName()));
+            if (!Collections.disjoint(wordsInName, upperCaseKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Transform all the strings in a collection to upper case and return them in a Set
```

Comment:
Please take note of the header comments' convention as given in our style guide.